parachains/runtimes/assets/statemine/src/lib.rs

```diff
@@ -450,6 +450,11 @@ impl parachain_info::Config for Runtime {}
 
 impl cumulus_pallet_aura_ext::Config for Runtime {}
 
+parameter_types! {
+	// Fellows pluralistic body.
+	pub const FellowsBodyId: BodyId = BodyId::Technical;
```

Comment:
I think that as long as the Tech Committee and Fellowship coexist, the Tech Comm would be able to control this origin.
https://github.com/paritytech/polkadot/pull/6273/files#r1049712864

Reply:
Right, but this should be valid?
Tech Committee has a root permissions.

Reply:
No, they do not. Only fast tracking permission.

Reply:
Double checked, Technical xcm body not used/mapped right now in kusama or polkadot (only docs tells as its supposed to be used as Tech Comm), it's safe to start using it as Fellows.